apps/spark/src/main/java/com/linkedin/openhouse/jobs/util/SparkJobUtil.java

```diff
@@ -20,10 +20,10 @@ private SparkJobUtil() {}
            "granularity": "DAY",
            "columnPattern": null }}
   Partitioned by time on datePartition column
-  Query: date_trunc('DAY', datePartition) < date_trunc('DAY', current_timestamp() - INTERVAL 30 DAYs)"
+  Query: datePartition < date_trunc('DAY', current_timestamp() - INTERVAL 30 DAYs)"
   */
   private static final String RETENTION_CONDITION_TEMPLATE =
-      "date_trunc('%s', %s) < date_trunc('%s', current_timestamp() - INTERVAL %d %ss)";
+      "%s < date_trunc('%s', current_timestamp() - INTERVAL %d %ss)";
```

Comment:
it's an equivalent change because trunc(x) < n and x < n are equivalent when x - real, n - integer

Reply:
do we have a unit test for the effect of this template ? 
IIUC the unit test below is about the format itself

Reply:
https://github.com/teamurko/openhouse/blob/ab478181442a59403b4a6a7c52b7879201e8e9a2/apps/spark/src/test/java/com/linkedin/openhouse/jobs/spark/OperationsTest.java#L39